spring-xsuaa/src/main/java/com/sap/cloud/security/xsuaa/token/XsuaaToken.java

```diff
@@ -248,17 +248,6 @@ public String getAppToken() {
 		return scopesList != null ? scopesList : Collections.emptyList();
 	}
 
-	/**
-	 * Check if the authentication token contains a claim, e.g. "email".
-	 *
-	 * @param claim
-	 *            name of the claim
-	 * @return true: attribute exists
-	 */
-	public boolean hasClaim(String claim) {
```

Comment:
why deleted? this method is used in line 158

Reply:
Oh instead hasClaim is used from the super class (Jwt i guess). I had it to fix the Stackoverflow Exception described in #528